hack/verify-vendor.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+#
+#   Copyright The containerd Authors.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+tmpdir="$(mktemp -d)"
+trap "rm -rf ${tmpdir}" EXIT
+
+git clone "https://github.com/containerd/project" "${tmpdir}"
+"${tmpdir}"/script/validate/vendor
+unset tmpdir
```

Comment:
If you unset `tmpdir` here, does trap still work?

Reply:
Hm, it still works, interesting.

We don't need to unset it actually, but it works anyway. :)

Reply:
:) I use `"` instead of `'`. I can remove the `unset` for the case, since it confuses.